addons/intercom_phone/functions/fnc_activate.sqf

```diff
@@ -0,0 +1,73 @@
+#include "script_component.hpp"
+
+/*
+  Name: TFAR_intercom_phone_fnc_activate
+
+  Author: Arend(Saborknight)
+    Connects the player to the intercom of a vehicle, if intercom is enabled on
+    the vehicle.
+
+  Arguments:
+    0: Vehicle object <OBJECT>
+    1: Player unit <OBJECT>
+
+  Return Value:
+    None
+
+  Example:
+    [_vehicle, _player] call TFAR_intercom_phone_fnc_activate;
+
+  Public: Yes
+*/
+
+params ["_vehicle", "_player"];
+
+private _phoneSpeaker = _vehicle getVariable ["TFAR_IntercomPhoneSpeaker", nil];
+
+if (!isNil "_phoneSpeaker") exitWith {
+    // [parseText format["<br /><br /><t align='center'>%1 is currently on the phone.</t>", name _phoneSpeaker]] call TFAR_fnc_showHint;
+    "no";
+};
+
+private _vehicleID = _vehicle getVariable ["TFAR_vehicleIDOverride", netid _vehicle];
+_vehicle setVariable ["TFAR_vehicleIDOverride", _vehicleID, true];
```

Comment:
Is this needed? considering you just read that variable and know that (atleast locally) it already has that value?